siff-dr-router/siff-header.go

```diff
@@ -0,0 +1,328 @@
+package main
```

Comment:
We should probably refactor this so we're not copy pasting the same file over and over